tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test if the Screen Sharing GNOME Desktop functionality works
+#   We currently need to install vino package for that
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    # Run the gnome-control-center - the sharing section
+    x11_start_program("gnome-control-center sharing", target_match => 'gnome-control-center');
+
+    # It may happen that the screen sharing is not available
+    if (check_screen 'without_screensharing') {
```

Comment:
Maybe I did not make myself clear so my proposal is the following:

```perl
assert_screen [qw(with_screensharing without_screensharing)];
if (match_has_tag 'without_screensharing') }
    …
}
assert_screen 'with_screensharing';
```

Reply:
This make sense now.